views/ok.ejs

```diff
@@ -1,35 +0,0 @@
-<html>
-<head>
-    <meta charset="utf-8">
-    <link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
-    <script type="text/javascript" src="bower_components/jquery/dist/jquery.min.js" charset="utf-8"></script>
-
-</head>
-<body>
-<script type="text/javascript" src="http://localhost:3000/bundle.js" charset="utf-8">
-</script>
-<script>
-    var ludwig = new Ludwig();
-</script>
-
-</a>
-</body>
-
-<body>
-
-<!-- Begin page content -->
-<div class="container">
-    <h1>Merci!</h1>
-    <p>Votre suggestion a bien été enregistrée, elle est accessible en suivant <a href="<%=pullRequestURL%>">ce lien</a></p>
```

Comment:
In my opinion that view should be in the application itself. Here are some context.

## Currently

With GitHub authentication:
From application contribution view,
- kick off a full page POST to Ludwig,
- receive a 302 redirect to GitHub (required by GitHub to make the authentication validation), then
    - if the logged-in user already accepted the application, 302 redirect to Ludwig view or
    - if the logged-in user hasn't accepted the application yet, show an acceptation request view.
- Then, back in Ludwig, chat with GitHub (create branch, add a commit, create a PR) and
- if everything works show the OK page.

In my opinion, the last step should be redirect to a specific page on the application with some details about the contribution (ie. URL, PR number, etc.)

Why ? Because it is a way to mimic an API workflow.


In my opinion, and as a first version (at least), this workflow should be updated to something like:
From the application view
- kick of an AJAX request to Ludwig that chats with GitHub (create branch, add a commit, create a PR) and
- if everything works return GitHub new PR API response.